Raspi/Test/mock_wireless_module.py

```diff
@@ -64,98 +109,55 @@ def send_fake_data(client, duration, rate, module_id):
-                                    "value": s_temperature.get_value()
-                                },
-                                {
-                                    "type": "humidity",
-                                    "value": s_humidity.get_value()
-                                },
-                                {
-                                    "type": "steeringAngle",
-                                    "value": s_steering_angle.get_value()
-                                }
-                             ]
-                          }
-            publish_data_and_battery(module_num)
-
-        # Wireless module 2 (Back)
-        if module_id == 2 or module_id is None:
-            module_num = 2
-            module_data = {
-                            "sensors": [
-                                {
-                                    "type": "co2",
-                                    "value": s_co2.get_value()
-                                },
-                                {
-                                    "type": "temperature",
-                                    "value": s_temperature.get_value()
-                                },
-                                {
-                                    "type": "humidity",
-                                    "value": s_humidity.get_value()
-                                },
-                                {
-                                    "type": "accelerometer",
-                                    "value": s_accelerometer.get_value()
-                                },
-                                {
-                                    "type": "gyroscope",
-                                    "value": s_gyroscope.get_value()
-                                }
-                             ]
-                          }
-            publish_data_and_battery(module_num)
-
-        # Wireless module 3 (Front)
-        if module_id == 3 or module_id is None:
-            module_num = 3
-            module_data = {
-                            "sensors": [
-                                {
-                                    "type": "co2",
-                                    "value": s_co2.get_value()
-                                },
-                                {
-                                    "type": "reedVelocity",
-                                    "value": s_reed_velocity.get_value()
-                                },
-                                {
-                                    "type": "gps",
-                                    "value": s_gps.get_value()
-                                }
-                             ]
-                          }
-            publish_data_and_battery(module_num)
+        print("TIME:", current_time)
+        for module_id_num in module_id_nums:
+            # Wireless module 1 (Front)
+            if module_id_num == 1:
```

Comment:
If you declared the modules in a list or dictionary you wouldn't need this lookup anymore